src/main/perf/LineFileDocs.java

```diff
@@ -615,7 +620,9 @@ public Document nextDoc(DocState doc) throws IOException {
       }
     }
 
-    final int myID = nextID.getAndIncrement();
+    if (myID == -1) {
```

Comment:
Can we remove this check since the `getNextId` method will throw if it is called too many times?

Reply:
Let's keep it?
I added the check because I declared the `myID` far above instead and want to make sure it is actually being assigned  in both branches, keep it would catch mistakes if in the future someone mistakenly removed the assignment?

Reply:
OK that logic makes sense -- let's keep it!